### ISSUE TITLE
Failed to deploy nighlty Eclipse Che using olm

### ISSUE THREAD

**Issue by tolusha** [eclipse-che/che#18532]
### Describe the bug
Failed to deploy nightly Eclipse Che using olm.
Olm creates clusterroles for `che-namespace-editor` and the name isn't the same as defined in the CR. for instance:

```
eclipse-che-preview-openshift.v7.23.0-36.nightly-xdnnm-chetl8fl                   55s
eclipse-che.v7.22.0-bbsj2-che-namespace-editor-jb2wn                              2m24s
```

### Che version
 - [x] nightly

### Steps to reproduce
Follow this doc 
[1] https://github.com/eclipse/che-operator/blob/master/olm/README.md#testing-custom-catalogsource-and-bundle-images-on-the-kubernetes

### Expected behavior
Eclipse Che is deployed

### Runtime
   - [x] Openshift (include output of `oc version`)
   - [x] minikube (include output of `minikube version` and `kubectl version`)

### Installation method
   - [x] olm

### Environment
   - [x] my computer
       - [x] Linux
   - [x] Cloud
       - [x] Amazon

### Operator logs
```
time="2020-12-04T10:46:03Z" level=info msg="Waiting on cluster role binding 'che-che-che-namespace-editor' to be created"
time="2020-12-04T10:46:03Z" level=error msg="clusterroles.rbac.authorization.k8s.io \"che-namespace-editor\" not found"
```

**Comment by sparkoo** [comment]
The default `cheClusterRoles` does not work on olm, because it names the clusterroles in it's own way (something like `eclipse-che-preview-openshift.v7.23.0-36.nightly-765b5b5886`) so it's completely different from default value. We need to figure out how to get the clusterrole name into CheCluster CR `server.cheClusterRoles` value.

cc: @davidfestal ?

So far I've created an PR removing the default value https://github.com/eclipse/che-operator/pull/565 which unblock the deployment, but the clusterrole won't be assigned to `che` serviceaccount, which will make namespace labeling not working.

**Comment by sparkoo** [comment]
@tolusha can you please confirm that https://github.com/eclipse/che-operator/pull/565 fixed the deploy issue? I want to create a new issue for olm installer setting `server.cheClusterRoles`

**Comment by tolusha** [comment]
@sparkoo 
yes, I confirm.

**Comment by sparkoo** [comment]
@tolusha thanks, Closing this one and I've created new one for olm installer setting `server.checlusterRoles` https://github.com/eclipse/che/issues/18544
